SetFlags.cmake

```diff
@@ -266,11 +266,18 @@ macro(set_exe_flags)
 				add_flags_cxx("-Wno-documentation")
 			endif()
 			if ("${CLANG_VERSION}" VERSION_GREATER 3.5)
+				include(CheckCXXCompilerFlag)
 				check_cxx_compiler_flag(-Wno-reserved-id-macro HAS_NO_RESERVED_ID_MACRO)
+				check_cxx_compiler_flag(-Wno-documentation-unknown-command HAS_NO_DOCUMENTATION_UNKNOWN)
 				if (HAS_NO_RESERVED_ID_MACRO)
 					# Use this flag to ignore error for a reserved macro problem in sqlite 3
 					add_flags_cxx("-Wno-reserved-id-macro")
 				endif()
+				if (HAS_NO_DOCUMENTATION_UNKNOWN)
+					# Ignore another problem in sqlite
+					add_flags_cxx("-Wno-documentation-unknown-command")
+				endif()
+				add_flags_cxx("-Wno-format-pedantic")
```

Comment:
C++ is a programming language based off C, no?

If UggBoots are the case, then it is of course necessary to fix those instances. I shall inform #2441 to remove the flag after all.